app/Http/Controllers/TeamController.php

```diff
@@ -105,11 +112,19 @@ public function destroy(Team $team) {
 	 * @param array $attributes
 	 * @return \App\Course
 	 */
-	private function createTeam(array $attributes) {
-		return Team::create([
+	private function createTeam(array $attributes, $image) {
+		$team = Team::create([
```

Comment:
Do we envision using this function anywhere else? Why did we pull it out?

Reply:
Decidí hacer una función a parte para que el código fuera más legible y previendo que `store()` crecerá aún más cuando se implementen otros user stories.